core/channels.config.ts

```diff
@@ -0,0 +1,17 @@
+import { type LocaleType } from './i18n';
+
+export type RecordFromLocales = {
+  [K in LocaleType]: string;
+};
+
+// Set overrides per locale
+const localeToChannelsMappings: Partial<RecordFromLocales> = {
+  // es: '12345',
+};
+
+function getChannelIdFromLocale(locale?: string) {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  return localeToChannelsMappings[locale as LocaleType] ?? process.env.BIGCOMMERCE_CHANNEL_ID;
```

Comment:
Returning `process.env.BIGCOMMERCE_CHANNEL_ID` until we can figure out how to programatically detect if `getLocale` won't work in `getChannelId`.